fn/result.go

```diff
@@ -77,15 +77,29 @@ func (r Result[T]) MapErr(f func(error) error) Result[T] {
 }
 
 // Option returns the success value as an Option.
+//
+// Deprecated: Use Hush instead.
 func (r Result[T]) Option() Option[T] {
 	return r.Either.LeftToOption()
 }
 
+// Hush mutes the error value of the result.
+func (r Result[T]) Hush() Option[T] {
```

Comment:
`ToOption`? `Hush` doesn't intuitively tell me what we're doing here

Reply:
If a second opinion agrees, I'll change it.